MBBSEmu/DOS/Interrupts/IInterruptHandler.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace MBBSEmu.DOS.Interrupts
+{
+    public interface IInterruptHandler
+    {
+        ushort Vector { get; }
```

Comment:
aren't there only 256 max?

Reply:
Yep -- good catch, I'm pulling the 'Immediate8' off the opcode for `INT`. I'll update the signature.